libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts

```diff
@@ -316,6 +316,10 @@ export class RecordedDashboards implements IWorkspaceDashboardsService {
         throw new NotSupported("recorded backend does not support this call");
     }
 
+    public deleteScheduledMail(_ref: ObjRef): Promise<void> {
+        return Promise.resolve();
```

Comment:
why is it different that the other ones around? (`throw new NotSupported("recorded backend does not support this call");`)

Reply:
I actually tried to follow the ones around - see https://github.com/gooddata/gooddata-ui-sdk/blob/4cfc945111f4ef505d5104896cf349d4788a4e44/libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts#L323-L329

But I will replace it as you suggest, because I'm not sure about its use-case..